formats/czi-extract-plugin/tests/fixture.py

```diff
@@ -0,0 +1,44 @@
+"""Test fixtures.
+
+Set up all data used in tests.
+"""
+import shutil
+import tempfile
+from pathlib import Path
+from typing import Union
+
+import pytest
+import requests
+
+
+def clean_directories() -> None:
+    """Remove all temporary directories."""
+    for d in Path(".").cwd().iterdir():
+        if d.is_dir() and d.name.startswith("tmp"):
+            shutil.rmtree(d)
```

Comment:
See the comment below.

Reply:
I am downloading large czi files and dont want to download it again for every test. which is why I am cleaning directories after running tests